File: week04/week04-1.cpp
//week04-1.cpp
//LeetCode 2529.Maximun Count of Postive Integer and Negative Integer
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int pos = 0,neg = 0;
        for( int i=0; i <nums.size(); i++) {
            if(nums[i] > 0 ) pos++;//正數++
            if(nums[i] > 0 ) neg++;//負數++
    }
    if( pos > neg) return pos; //正數比較大 送出去
    else return neg;//不然就負數比較大 送出去
    }
};

File: week04/week04-1.py
#week04-1.py
#LeetCode 2529.Maximun Count of Postive Integer and Negative Integer
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0
        for i in range(len(nums)):
            if nums[i] > 0: pos += 1 #正數
            if nums[i] < 0: neg += 1 #負數
        return max(pos, neg) #把大的數 return 傳回去
